src/interfaces/interfaces.js

```diff
@@ -54,6 +54,16 @@ export type EditFormActionData = {
     priority: Priority
 }
 
+export type AddFormActionData = {
+    id: number,
+    issue: string,
+    priority: Priority,
+    assignee: {
```

Comment:
NIT: assignee could be a separate type

Reply:
yep, just like Priority...
ok, I'll change it